torch/onnx/symbolic_opset14.py

```diff
@@ -117,3 +119,174 @@ def quantized_hardswish(g: jit_utils.GraphContext, x, op_scale, op_zero_point):
     output = hardswish(g, x)
 
     return symbolic_helper.quantize_helper(g, output, op_scale, op_zero_point)
+
+
+# Ported from https://github.com/microsoft/onnx-script/blob/main/onnxscript/function_libs/torch_aten/ops/core.py
```

Comment:
Prefer a permanent link so it doesn't expire when we refactor

Reply:
You can get it from here 
![image](https://user-images.githubusercontent.com/11205048/233499810-20ec923a-f981-4f3e-b888-833d6cf1ff9c.png)

Reply:
https://github.com/microsoft/onnxscript/blob/c451f9f7302ec607d4d897aeef2dde902d880cef/onnxscript/function_libs/torch_aten/ops/nn.py#L1505

Reply:
Done